etcdctl/README.md

```diff
@@ -31,6 +31,8 @@ RPC: Put
 
 - ignore-value -- updates the key using its current value.
 
+- ignore-lease -- updates the key using its current lease
```

Comment:
Sure! Let me know if anything else is required or the rest is fine. Will do all the changes together in one shot. Thanks!